modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -10,6 +10,8 @@ NOTE: {org-name} recommends running OpenShift using the method described in this
 == Prerequisites
 
 * MacOS or Linux
++
+NOTE: Due to an link:https://github.com/openshift/origin/issues/18747[OpenShift bug], {product-name} will not run on RHEL/CentOS 7.4 or RHEL/CentOS 7.5.
 
```

Comment:
It is fine for me and it is regarded to the issue https://github.com/aerogear/mobile-core/issues/194#issuecomment-408899008

Reply:
if above is true, then we shouldn't mention RHEL elsewhere in doc